src/scripts.js

```diff
@@ -8,8 +8,15 @@ import {
 } from './api-calls.js';
 import Destinations from './Destinations';
 import Trips from './Trips';
+const dayjs = require('dayjs')
+let RelativeTime = require('dayjs/plugin/RelativeTime')
+dayjs.extend(RelativeTime)
+
+console.log(dayjs().diff('2022/03/01', 'day'))
```

Comment:
It's fun that dayjs recognizes this date format even though it prefers YYYY-MM-DD